src/app/store/telemetry-search.js

```diff
@@ -1,37 +1,22 @@
 import { readable } from 'svelte/store';
 
 import FlexSearch from 'flexsearch'
-const url = 'https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes'
+// TODO: Make this dynamic based on prod vs local dev.
```

Comment:
What are you thoughts on achieving this? Is there a pattern for this?

Reply:
it seems you've actually taken care of this, right? `make build` should handle the cors stuff for this docker instsance

Reply:
ah, actually. I would just use `/api/v1/probes`. That handles everything you need.

Reply:
Using `/api/v1/probes` wouldn't handle the `:8000` (front-end) to `:5000) (back-end) host change. I imagine some set of constants defined for a locally running instance vs a production running instance.